database/src/main/kotlin/no/nav/su/se/bakover/database/søknad/SøknadRepoInternal.kt

```diff
@@ -45,6 +45,19 @@ internal object SøknadRepoInternal {
         }
         return KunneIkkeAvslutteSøknadsBehandling.left()
     }
+
+    fun finnesBehandlingForSøknadInternal(
+        søknadId: UUID,
+        session: Session
+    ): Boolean {
+       val finnesBehandlingForSøknad = "select * from behandling where søknadId=:id".hent(
```

Comment:
Kunne tatt id istedet for *. Antar kanskje count ikke er så lett med det vi har foreløpig?